### DISCUSSION TITLE
ABOUT  SUBTITLES IN UPCRAFT VIDEO

### DISCUSSION THREAD

**Discussion by 48cork** [Cyfrin/foundry-full-course-cu#2408] [General]
Why I dont have subtitles in upcraft videos?

Thanks in advance

**Comment by EngrPips** [comment]
Hello @48cork, Please join the official discord of Cyfrin https://discord.gg/t8qVHJCb and submit a ticket #platform issues for this issue you run into.

**Comment by 48cork** [comment]
I gott the issue fixed, thanks

Em sex., 6 de set. de 2024 04:34, EngrPips ***@***.***>
escreveu:

**Reply by EngrPips** [reply]
You are welcome.
